docs/installation-guide-mac.rst

```diff
@@ -0,0 +1,62 @@
+:orphan:
+
+Installation guide for macOS
+============================
+
+Follow these instructions to install the CityEnergyAnalyst (CEA) on a macOS system (tested with Mojave 10.14.5) from the source
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Download and install Git (64-bit).
+* Download and install Github Desktop (64-bit).
+* Download and install Miniconda(64-bit) for Python 2.7.
+* Download and install CMake.

```

Comment:
tested with ``cmake-3.15.0-Darwin-x86_64.dmg`` but newer should work too

Reply:
btw: for `cmake` to be available in the terminal, you need to add `export PATH="/Applications/CMake.app/Contents/bin":"$PATH"` to `.bash_profile`.

Reply:
or... just type it before the cmake call (then it's not permanent... i like this better!)